.github/workflows/build_SRWA.yml

```diff
@@ -0,0 +1,89 @@
+# This workflow Builds the UFS SRWEATHER APP
+name: Build SRWA
+
+on:
+  push:
+    branches: [ feature/CI_build_test ]
+  pull_request:
+    branches: [ rrfs_ci, feature/CI_build_test ]
+
+# Use a default login shell which loads intel/oneapi/setvars.sh, and lmod path 
+defaults:
+  run:
+    shell: bash -l {0}
+
+jobs:
+ 
+  Clone_rrfs_ci_pcluster:
+    # Auxillary repository needed to insert env files to run CI tests in the short term 
+    name: Clone RRFS CI repo
+    # This job will run on AWS Parallel Cluster head node
+    runs-on: self-hosted
+    steps:
+    
+    - name: Checkout robgonzalezpita/rrfs-ci-pcluster Repo
+      uses: actions/checkout@v3
+      with:
+        repository: robgonzalezpita/rrfs-ci-pcluster
+        path: ./rrfs-ci-pcluster
+  
+  Configure_SRWA:
+    name: Checkout SRWA in parent directory 
+    runs-on: self-hosted 
+    needs: [ Clone_rrfs_ci_pcluster ]
+
+    steps:
+      # Checkout the SRWA - in parent directory, with commit sha-1 in filepath to avoid multiple PR Conflicts, & track builds by commit
+      # Specify branch used for testing, if there is not a similar named branch in ufs-SRWA
+      - name: Checkout SRWA 
+        run: |
+          cd ../
```

Comment:
We will likely want a bit more information about the directory structure here. Consider using full paths, or a doc block to explain the topology that we'll be traversing here. Please address the relative paths in the next sections similarly.

Reply:
The full path used will be dependent on the location we install the self hosted runner. I'm holding off on this one for now until we get the NOAA-GSL self-hosted runner configured.

Reply:
I've added a couple of comments addressing the reason why relative paths are used and the topology we're traversing.